docs/tutorial/installation.rst

```diff
@@ -11,21 +11,21 @@ Installing Python
 =================
 
 There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
-That's because we will create a new Python environment.
-As well as containing a Python installation, this environment provides an isolated place to install Python packages (like ``sunpy``) without affecting any other current Python installation.
+That's because we will create a new Python virtual environment.
+As well as containing a Python installation, this virtual environment provides an isolated place to install Python packages (like ``sunpy``) without affecting any other current Python installation.
 If you already have Python and ``conda`` working you can skip the next section.
 
 `If you are using Anaconda, we recommend that you uninstall it as the default package channel(s) have a restrictive license which means you might not be able to use it for free <https://sunpy.org/posts/2024/2024-08-09-anaconda/>`__.
-Instead, we recommend that you use miniforge which is a minimal installer that setups conda with the conda-forge channel, which is free to use for everyone.
+Instead, we recommend that you use miniforge which is a minimal installer that setups ``conda`` with the ``conda-forge`` channel, which is free to use for everyone.
 If you are using miniforge, you can skip the next section
 
 .. _sunpy-tutorial-installing-miniforge:
 
-Installing miniforge
---------------------
+Installing miniforge (and conda)
+================
```

Comment:
The under title has to match the length of the title

Reply:
Thanks for the feedback! I have updated the subtitle length to match the title as requested please let me know if there's anything else that needs adjustment